src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx

```diff
@@ -284,6 +287,31 @@ export default function MeasureInformation() {
             />
           </div>
 
+          <br />
+          <DisplayDiv>
+            <span className={classes.asterisk}>*</span>
+            <span tw="mr-2" className={classes.fieldName}>
+              ECQM Abbreviated Title:
+            </span>
+          </DisplayDiv>
+          <div className={formRowGapped}>
+            <TextField
```

Comment:
Can we use components from madie-design-systems?

Reply:
Thanks, will do!

Reply:
Thanks, will do!